Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  Calculator - ViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+class CalculatorViewController: UIViewController {
+    // MARK: - IBOutlet
+    
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var calculateItemsStack: UIStackView!
+    @IBOutlet private weak var operatorLabel: UILabel!
+    @IBOutlet private weak var operandLabel: UILabel!
+    
+    // MARK: - Property
+    
+    private var components: String = ""
+    
+    // MARK: - View State Method
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        clearAll()
+    }
+    
+    // MARK: - IBAction
+    
+    @IBAction private func tapACButton(_ sender: Any) {
+        clearAll()
+    }
+
+    @IBAction private func tapCEButton(_ sender: Any) {
+        clearEntry()
+    }
+
+    @IBAction private func tapNEGButton(_ sender: Any) {
+        toggleNegativeNumber()
+    }
+
+    @IBAction private func tapOperatorButton(_ sender: UIButton) {
+        guard !components.isEmpty || (operandLabel.text != "0") else {
+            return
+        }
+        
+        guard formatNumber(of: Decimal(string: operandLabel.text ?? "")) != "0" else {
+            operatorLabel.text = sender.titleLabel?.text
+            return
+        }
+        
+        appendCalculateItem()
+        
+        operatorLabel.text = sender.titleLabel?.text
+        clearEntry()
+    }
+
+    @IBAction private func tapNumberButton(_ sender: UIButton) {
+        let operandLabelText = removeComma(of: operandLabel.text)
+        
+        guard var operandLabelText, operandLabelText.count < 20 else {
+            return
+        }
```

Comment:
```suggestion        
        guard var operandLabelText = removeComma(of: operandLabel.text), 
             operandLabelText.count < 20 
        else {
            return
        }
```
이렇게 수정되는 건 어떨까요?
개인적으로 일반적인 optional binding 방법이 아니라 문법적으로 이해하는데 "어?" 하는 포인트가 있었어요.😂
모든 사람들이 이해할 수 있는 일반적인 문법이 가독성이 높다는 생각이 드는데...
비모는 어떻게 생각하시나요!

Reply:
생략해도 되는 부분이었는데 제대로 확인하지 못하고 지나갔네요!
제안주신 코드로 변경했습니다 👍